mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/concurrency/ConcurrentTestRunnerTest.java

```diff
@@ -120,18 +118,18 @@ public void consume(int threadNumber, int step) throws Exception {
     public void runShouldPerformAllOperations() throws Exception {
```

Comment:
The aim is to do some concurrent testing. You start x threads at the same time, and perform operations (parametrized by thread number and operation number). This allow you to detect some troubbles.

Here I introduced it to test setFlags when the lightweigth condition (optimistic transaction) fails and some retry was needed.